storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.11.rst

```diff
@@ -0,0 +1,23 @@
+===========================
+|Percona XtraBackup| 2.4.11
+===========================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.4.11 on
+April 23, 2018. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.11/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is based on `MySQL 5.7.19
+<http://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-19.html>`_
+and is the current GA (Generally Available) stable release in the 2.4 series.
+
+Starting from now, |Percona XtraBackup| issue tracking system was moved from
+launchpad to `JIRA <https://jira.percona.com/projects/PXB>`_.
+All Percona software is open-source and free.
+
+New features and improvements
+=============================
+
+* The support of the |Percona Server| `encrypted tablespace <https://www.percona.com/doc/percona-server/5.7/management/data_at_rest_encryption.html#id7>`_ was implemented in this version of |Percona XtraBackup|. Issue fixed :jirabug:`PXB-1513`.
+  
+* |Percona XtraBackup| also supports now `keyring_vault plugin <https://www.percona.com/doc/percona-server/5.7/management/data_at_rest_encryption.html#id13>`_ of |Percona Server|. Issue fixed :jirabug:`PXB-1514`.
```

Comment:
Percona Xtrabackup is now able to backup encrypted PS instances which are using keyring vault plugin.